### PR TITLE
Add bootstrap for Certification

### PR DESCRIPTION
**Pull request by thuyvy3110**
Separate layout between Item & Info by row

### CODE REVIEW

File: views/components/Certification.js [hunk 1]
```diff
@@ -68,6 +61,4 @@ const List = styled.ul`
  * Note: This is the workaround solution, because this section hasn't been implemented with bootstrap
```
Comment (DeKal) [inline views/components/Certification.js]: Remove this

File: views/components/Certification.js [hunk 2]
```diff
@@ -68,6 +61,4 @@ const List = styled.ul`
...
  * Refer to this issue: https://github.com/DeKal/portfolio-next-js/issues/78
  */
-const WorkAroundContainer = styled(Container)`
-  height: 600px;
-`
+const WorkAroundContainer = styled(Container)``
```
Comment (DeKal) [inline views/components/Certification.js]: Remove this

File: views/components/Certification.js [hunk 3]
```diff
@@ -1,57 +1,50 @@
 import React, { useState } from 'react'
 import styled from 'styled-components/macro'
-import Cert from '~/views/components/certification/Cert'
+import CertItem from '~/views/components/certification/CertItem'
+import CertInfo from '~/views/components/certification/CertInfo'
 import { CERTIFICATIONS } from '~/consts/pages'
-import { CERT_ACTIVE_ITEM } from '~/consts/certification'
+import { ITEM, INFO } from '~/consts/certification'
 import Container from '~/views/components/common/section/Container'
```
Comment (DeKal) [inline views/components/Certification.js]: `import SectionContainer from '~/views/components/common/section/Container'`

File: views/components/Certification.js [hunk 4]
```diff
@@ -1,57 +1,50 @@
...
 import Anchor from '~/views/components/common/Anchor'
+import Row from 'react-bootstrap/Row'
+import BContainer from 'react-bootstrap/Container'
 
 const Certification = () => {
-  const [itemSelected, setSelectedItem] = useState(CERT_ACTIVE_ITEM[0])
+  const [itemSelected, setSelectedItem] = useState(1)
 
   return (
     <WorkAroundContainer>
```
Comment (DeKal) [inline views/components/Certification.js]: `SectionContainer `

File: views/components/Certification.js [hunk 5]
```diff
@@ -1,57 +1,50 @@
...
       <Anchor id={CERTIFICATIONS} top={-40} />
-      <div className="container">
+      <BContainer>
```
Comment (DeKal) [inline views/components/Certification.js]: Just `Container `

File: views/components/certification/CertInfo.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+import Info from '~/views/components/certification/Info'
+import Title from '~/views/components/certification/Title'
+import { LEFT_INFO_POS } from '~/consts/certification'
+import PropTypes from 'prop-types'
+import { Translate } from 'react-localize-redux'
+import Col from 'react-bootstrap/Col'
+
+const CertInfo = ({ itemSelected, id, title, content }) => {
+  return (
+    <StyleCol sm={12} xs={12} itemSelected={itemSelected} id={id}>
```
Comment (DeKal) [inline views/components/certification/CertInfo.js]: can use  better name like `ShowOnSelectedCol`

File: views/components/certification/CertItem.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import CertIcon from '~/views/components/certification/CertIcon'
+import Item from '~/views/components/certification/Item'
+import PropTypes from 'prop-types'
+import Col from 'react-bootstrap/Col'
+
+const CertItem = ({ itemSelected, setSelectedItem, id, name }) => {
+  return (
+    <Col sm={3} md={3} xs={3} lg={3}>
```
Comment (DeKal) [inline views/components/certification/CertItem.js]: xs sm is enough

**Comment by DeKal** [comment]
Remove proccess.sass also

File: views/components/Certification.js (hunk 3 above)
Comment (thuyvy3110) [reply views/components/Certification.js]: Updated

File: views/components/Certification.js (hunk 4 above)
Comment (thuyvy3110) [reply views/components/Certification.js]: Updated

File: views/components/certification/CertInfo.js (hunk 6 above)
Comment (thuyvy3110) [reply views/components/certification/CertInfo.js]: Updated

File: views/components/certification/CertItem.js (hunk 7 above)
Comment (thuyvy3110) [reply views/components/certification/CertItem.js]: Updated

File: views/components/Certification.js (hunk 5 above)
Comment (thuyvy3110) [reply views/components/Certification.js]: Updated

File: views/components/Certification.js (hunk 3 above)
Comment (thuyvy3110) [reply views/components/Certification.js]: Updated

File: views/components/Certification.js (hunk 2 above)
Comment (thuyvy3110) [reply views/components/Certification.js]: Updated

File: views/components/Certification.js (hunk 1 above)
Comment (thuyvy3110) [reply views/components/Certification.js]: Updated

**Pull request merged**